### DISCUSSION TITLE
Updating

### DISCUSSION THREAD

**Discussion by imba4** [DISTRHO/Cardinal#322] [Q&A] [answered]
sorry for the probably stupid question, how do i update the software? I have to re-download the whole form or are there specific.
i am completely new to this kind of programs and i don't know anything about programming :(
sorry again and thanks for your time

**Comment by falkTX** [comment, marked as answer]
yes, download new version and install that one. it will override the older version.

**Reply by imba4** [reply]
thank you so much.
thanks for your work
